cypress/e2e/cloud/userAccounts.test.ts

```diff
@@ -4,158 +4,87 @@ const doSetup = (cy, numAccounts: number) => {
   cy.flush().then(() => {
     cy.signin().then(() => {
       cy.get('@org').then(({id}: Organization) => {
+        cy.visit(`/orgs/${id}/accounts/settings`)
```

Comment:
UserAccounts required significant refactoring.

First, there's no longer any point in checking for the 'switch-account' button or doing tests around that. The switch-account functionality lives in globalheader now, so those tests are moot (and will fail when multiOrg is on, since the account page doesn't have a switch-account button in that case).

Second, userAccounts has been causing some pipeline issues for us because the tests do not have a clean interaction with our test environment. They modify the account state in quartz-mock and don't reset it, so when multiple tests are being run in parallel (or another test relying on account information happens to be run afterwards), it contaminates the environment, causing test failures down the line.

The better solution here is to dispense with using 'quartzProvision,' and simply use `cy.intercept` to mock the quartz get-account and rename-account APIs.